iml-action-runner.service

```diff
@@ -13,6 +13,7 @@ After=iml-action-runner.socket
 Type=simple
 Environment=RUST_LOG=info
```

Comment:
This takes priority over the files.

Reply:
It doesn't:

https://www.freedesktop.org/software/systemd/man/systemd.exec.html#EnvironmentFile=

> Settings from these files override settings made with Environment=.

Reply:
From https://www.freedesktop.org/software/systemd/man/systemd.exec.html#EnvironmentFile=:
> Settings from these files override settings made with Environment=. If the same variable is > set twice from these files, the files will be read in the order they are specified and the later > setting will override the earlier setting.

Reply:
Hm, I wasn't able to override that in a file...